lib/registrations/lanes/competing.rb

```diff
@@ -42,13 +42,17 @@ def self.update!(update_params, registration, current_user_id)
           changes[:event_ids] = registration.changed_event_ids if registration.changed_event_ids.present?
 
           registration.save!
-          registration.add_history_entry(changes, 'user', current_user_id, Registrations::Helper.action_type(update_params, current_user_id))
+          if acting_entity_id == 'auto-accept'
+            registration.add_history_entry(changes, 'System', acting_entity_id, Registrations::Helper.action_type(update_params, acting_entity_id))
```

Comment:
Where does this `'System'` come from and is it standardized somewhere?

Reply:
Is it now!